pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["wheel", "setuptools", "setuptools_scm"]
```

Comment:
@kotamanegi 
Hi, thank you so much for having interests in this project! 😄 
I am new to pyproject.toml and wondering if `"setuptools_scm"` is necessary?

Reply:
`setuptools_scm` is not so much necessary (in this project).
I've removed "setuptools_scm".

Reply:
Thanks!